README.md

```diff
@@ -29,6 +34,18 @@ To use this action in your workflow, add the following step configuration to you
     INTEGRATION_ID: <INTEGRATION_ID>
 ```
 
+or
+
+```yaml
+- name: <STEP NAME>
+  uses: prismatic-io/integration-publisher@v1.0
```

Comment:
Technically this wouldn't be valid in 1.0 since PATH_TO_CNI is a post-release addition

Reply:
That's true. I can update it with the expected new tag, or I can apply a <current_version> placeholder.

Reply:
I opted for LATEST_VERSION

Reply:
We can update the READme without having to draft a release, anytime we want to revise.